README.md

```diff
@@ -1 +1,111 @@
-iOS-DebugKit
+# DebugKit
```

Comment:
Another thing I might add to the README might be something about the dependency on MetricKit (pending whatever we may decide).

Also, should there be some info about what you'll get by default out of the box? Should there be some sort of "default" DebugView or configuration that can be dropped in that's pre-configured with the common sections as in the example below? That way in our client projects, the only real customization they would need to do would be to add things like Wormholy or fonts/colors/etc. onto the default configuration.

Reply:
I added some context about the dependencies to the `README`, along with some of the out of the box logging functionality.

I didn't create a `default` DebugView because it can vary so much (sections, content details), so I tried to use the example at the top of the `README` as a jumping off point for a debug menu's contents.